po/de.po

```diff
@@ -307,6 +303,34 @@ msgstr "Abbild mit Fingerabdruck %s importiert\n"
 msgid "Information about remotes not yet supported\n"
 msgstr "Informationen über entfernte Instanzen wird noch nicht unterstützt\n"
 
+#: lxc/init.go:22
+#, fuzzy
+msgid ""
+"Initialize a container from a particular image.\n"
+"\n"
+"lxc init [remote:]<image> [remote:][<name>] [--ephemeral|-e] [--profile|-p "
+"<profile>...]\n"
+"\n"
+"Initializes a container using the specified image and name.\n"
+"\n"
+"Not specifying -p will result in the default profile.\n"
+"Specifying \"-p\" with no argument will result in no profile.\n"
+"\n"
+"Example:\n"
+"lxc init ubuntu u1\n"
+msgstr ""
+"Starte Container von gegebenem Abbild.\n"
+"\n"
+"lxc launch <Abbild> [<Name>] [—ephemeral|-e] [—profile|-p <Profile>…]\n"
```

Comment:
I'm not familiar with these strings files, but this looks like the translation for lxc launch is being attached to the string for lxc init… and down at the end of this file, lxc init's original translation is commented out